sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -1237,6 +1247,7 @@ aggregates AS (
     SUM(
       scalar_parent_urlbar_impression_autofill_url
     ) AS scalar_parent_urlbar_impression_autofill_url_sum,
+    AVG(places_previousday_visits) AS places_previousday_visits_mean,
```

Comment:
[it seems like the original probe is a count](https://searchfox.org/mozilla-central/source/toolkit/components/places/PlacesDBUtils.sys.mjs#1146-1151). I think either an average, or a max, might be appropriate here.  It looks like previously we had places_bookmarks_count_mean and places_pages_count_mean, so an average would be consistent with that at least.